conda/conda-build/conda_build_config.yaml

```diff
@@ -2,9 +2,8 @@
 # so use an empty string "" to denote a falsey value
 gpu_enabled:
   - true
-  - 
+  -
```

Comment:
This was already here but not sure why, can it be removed entirely?

Reply:
^ @marcinz @m3vaz ?

Reply:
Line 1 and 2:
```
# Conda only accepts strings as variant configs
# so use an empty string "" to denote a falsey value
```

Reply:
The more you know :shrug: thanks @m3vaz